USAGE_desktop.md

```diff
@@ -33,6 +33,10 @@ Vulkan API calls on Desktop systems.
     4. [Trimmed File Optimizer](#trimmed-file-optimizer)
     5. [To ASCII](#to-ascii)
     6. [Command Launcher](#command-launcher)
+4. [Layer Details](#layer-details)
```

Comment:
These four new intra-document links don't work, can be tested by clicking them in  https://github.com/davidlunarg/gfxreconstruct/blob/david_auto_doc_gen/USAGE_desktop.md

Reply:
Will change this to link to Layer Options section at the end, which will contain a link to layer options in the SDK documentation.